### ISSUE TITLE
Alive icons cover over code lens testing icons

### ISSUE THREAD

**Issue by clarity99** [CodeConnect/AliveFeedback#120]
When you mouse over Alive icons and it expands to three icons in covers over code lens testing icon, so you cannot click it:
![image](https://cloud.githubusercontent.com/assets/123329/15140870/64630d4e-169d-11e6-8b68-9a438f6c844f.png)

**Comment by JoshVarty** [comment]
Sorry for the slow response.

Is this still a problem? What kind of file structure do you have here? Is it just a vanilla Namespace -> Class -> Method hierarchy?

**Comment by clarity99** [comment]
Yes, still the problem with the last version. Just normal structure for tests - as you said - namespace - class - method.

**Comment by clarity99** [comment]
VS 2015 update 2
